other-identifiers/src/test/java/org/reactome/release/otherIdentifiers/PostStepChecksTest.java

```diff
@@ -2,44 +2,100 @@
+    Collection<GKInstance> rgpInstanceCollection = new ArrayList<>();
+    List<String> otherIdentifierList = new ArrayList<>();
+    List<String> otherIdentifierList2 = new ArrayList<>();
 
     @Test
-    public void getOtherIdentifierCountReturnsAtomicInt() throws Exception {
+    public void getOtherIdentifierCountReturnsCorrectOtherIdentifierCount() throws Exception {
+
+        rgpInstanceCollection.add(mockRGPInstance);
+        rgpInstanceCollection.add(mockRGPInstance2);
+        otherIdentifierList.add("Mock otherIdentifier");
+        Mockito.when(mockRGPInstance.getAttributeValuesList(ReactomeJavaConstants.otherIdentifier)).thenReturn(otherIdentifierList);
+        Mockito.when(mockRGPInstance2.getAttributeValuesList(ReactomeJavaConstants.otherIdentifier)).thenReturn(otherIdentifierList2);
+        long otherIdentifierCount = PostStepChecks.getCountOfInstancesWithOtherIdentifiers(rgpInstanceCollection);
+        assertThat(otherIdentifierCount, is(equalTo(correctOtherIdentifierCount)));
+    }
+
+    @Test
+    public void instanceWithoutOtherIdentifiersReturnsFalse() throws Exception {
+        Mockito.when(mockRGPInstance.getAttributeValuesList(ReactomeJavaConstants.otherIdentifier)).thenReturn(otherIdentifierList);
+        boolean hasOtherIdentifiers = PostStepChecks.hasOtherIdentifiers(mockRGPInstance);
+        assertThat(hasOtherIdentifiers, is(equalTo(false)));
+    }
+
+    @Test
+    public void instanceWithOtherIdentifiersReturnsTrue() throws Exception {
+        rgpInstanceCollection.add(mockRGPInstance);
+        otherIdentifierList.add("Mock otherIdentifier");
+        Mockito.when(mockRGPInstance.getAttributeValuesList(ReactomeJavaConstants.otherIdentifier)).thenReturn(otherIdentifierList);
+        boolean hasOtherIdentifiers = PostStepChecks.hasOtherIdentifiers(mockRGPInstance);
+        assertThat(hasOtherIdentifiers, is(equalTo(true)));
+    }
 
-        gkInstanceCollection.add(mockGKInstance);
-        gkInstanceCollection.add(mockGKInstance);
-        otherIdentifierCollection.add("Mock otherIdentifier");
-        Mockito.when(mockGKInstance.getAttributeValuesList(ReactomeJavaConstants.otherIdentifier)).thenReturn(otherIdentifierCollection);
-        int otherIdentifierCount = PostStepChecks.getOtherIdentifierCount(gkInstanceCollection);
+    @Test
+    public void getCountOfInstancesWithOtherIdentifiersNullTest() {
+        try {
+            long otherIdentifierCount = PostStepChecks.getCountOfInstancesWithOtherIdentifiers(null);
+        } catch (Exception e) {
+            e.printStackTrace();
+            assertTrue(e.toString().contains("NullPointerException"));
+        }
+    }
+
+    @Test
+    public void hasOtherIdentifiersNullTest() {
+        try {
+            PostStepChecks.hasOtherIdentifiers(null);
+        } catch(Exception e) {
+            e.printStackTrace();
+            assertTrue(e.getMessage().equals("Unable to retrieve other identifiers from RGP instance null"));
+        }
```

Comment:
It might be better to use `assertThrows` from Hamcrest to check an expected exception.  I have an example that I've used in my code here:

https://github.com/reactome/data-release-pipeline/blob/develop/data-exporter/src/test/java/org/reactome/release/dataexport/EuropePMCTest.java#L90